desc/magnetic_fields.py

```diff
@@ -1803,6 +1810,375 @@ def from_surface(
         )
 
 
+class OmnigenousField(Optimizable, IOAble):
+    """Model field with perfect omnigenity.
+
+    Notes
+    -----
+    Doesn't conform to MagneticField API, as it only knows about :math:`|B|` in flux
+    coordinates, not vector B in lab coordinates.
+
+    """
+
+    _io_attrs_ = [
```

Comment:
missing NFP. Should also add an IO test for this class since it's something that will likely be saved/loaded frequently